firmata/readme.md

```diff
@@ -0,0 +1,46 @@
+# Usage
+
+For typescript projects, instead of
+
+```js
+var Board = require('firmata');
+```
+
+you should use
+
+```typescript
+import * as Board from 'firmata'
+```
+
+You will find that `const enums` are exported under the namespace `Board` for ease of comparison or
+use with other libraries before any boards are actually instantiated.
+
+It's also important to note that if you want to extend the `Board` class in typescript, the
+following will work (as long as you turn on `--noImplicitUseStrict` when compiling); here's
```

Comment:
So, it compiles fine, but at run-time when you actually try to extend a class, you end up getting an error (or I did at one point). I found that by removing the `"use strict"` declaration, this problem went away and it was able to run just fine. However, the structure has changed, and it may very well not be an issue anymore.

Is there a way I can test this before it gets published to `@types/firmata`? Some sort of linking procedure maybe? If its obvious, I apologize, this realm is all new to me.